Source/ACE.Server/WorldObjects/Player_Inventory.cs

```diff
@@ -1818,7 +1824,7 @@ private bool DoHandleActionStackableMerge(WorldObject sourceStack, Container sou
                 }
                 else if (sourceStackRootOwner != null)
                 {
-                    if (!sourceStackRootOwner.TryRemoveFromInventory(sourceStack.Guid, out _))
+                    if (!sourceStackRootOwner.TryRemoveFromInventory(sourceStack.Guid, out _) && (!(sourceStackRootOwner is Player playerContainer) || !playerContainer.TryDequipObjectWithNetworking(sourceStack.Guid, out _, DequipObjectAction.DequipToPack)))
```

Comment:
My only thought here is that we're doing TryRemoveFromInventory (without networking), but then TryDequipObject (WITH networking).

We have this situation often where we try both cases, but they should mostly (if not all) be either both with or both without. Stacks could be an exception though.

Reply:
i have no idea the proper fix for either of these bugs

all i know is there are tons of inventory/stackable/full mainpack bugs, and this fixes these particular ones

Reply:
Just verified that TryDequipObjectWithNetworking is the proper function to use here, and a regular dequip leads perma-buff vulnerabilities open

For bug 2, repro with Phials and TryDequipObjectWithBroadcasting instead, and you will be permabuffed forever.